components/scream/scripts/scream-env-cmd

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+"""
+Dump the SCREAM-approved environment setup commands for a machine
```

Comment:
Isn't this part of a different PR somewhere that Jim is doing?

Reply:
Ah, something must be off with the base branch. I'll rebase on master and force push.

Reply:
Ok, that seems to have fixed it.